gooddata-java-model/src/main/java/com/gooddata/sdk/model/settings/Settings.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2007-2021, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.sdk.model.settings;
+
+import com.fasterxml.jackson.annotation.*;
+import com.gooddata.sdk.common.util.GoodDataToStringBuilder;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import static com.gooddata.sdk.common.util.Validate.notEmpty;
+
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonTypeName("settings")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Settings implements Iterable<SettingItem> {
+
+    @JsonProperty("items")
+    private final List<SettingItem> items = new ArrayList<>();
+
+    @JsonCreator
+    public Settings(@JsonProperty("items") List<SettingItem> items) {
+        if (items != null && !items.isEmpty()) {
+            this.items.addAll(items);
+        }
+    }
+
+    @Override
+    public Iterator<SettingItem> iterator() {
+        return items.iterator();
+    }
+
+    /**
+     * Returns true if the setting item with given name exists and is enabled, false otherwise.
+     *
+     * @param settingName the name of setting item
+     * @return true if the setting item with given name exists and is enabled, false otherwise
+     */
+    public boolean isEnabled(final String settingName) {
```

Comment:
It would be also nice to have more general e.g. `String getValue(String settingKey)` (which can be reused by this one).

Reply:
Added